index.html

```
@@ -254,6 +254,49 @@ <h2 id="movie-list-title">지금 인기 있는 영화</h2>
         <p><img src="./images/woowacourse_logo.png" width="180" /></p>
       </footer>
     </div>
+
+    <div class="modal-background" id="modal-background">
+      <div class="modal">
+        <button class="close-modal" id="close-modal">
+          <img src="./images/modal_button_close.png" />
+        </button>
+        <div class="modal-container">
+          <div class="modal-image">
+            <img src="" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)